msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/usgov/arlington/TestCase938368.java

```diff
@@ -42,10 +48,13 @@
 import org.junit.Test;
 
 import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
 
 // Interactive token acquisition with instance_aware=true, login hint present, and federated account,
 // and WW common authority
 // https://identitydivision.visualstudio.com/Engineering/_workitems/edit/938368
+// Adding a retry on failure, sometimes arlington login page fails to load
```

Comment:
Does it give an error when it fails to load? How often does this happen? 1 in 5 times? 1 in 10 times? 1 in 20 times? or some other frequency?

Reply:
Sometimes gets the "This page isn't working" error. I'd say around 1 in 15 times it would happen. I also added code in this to refresh the page if the error shows up.